ophyd/quadem.py

```diff
@@ -16,7 +16,9 @@ def _current_fields(attr_base, field_base, range_, **kwargs):
 
     return defn
 
+
 class QuadEM(SingleTrigger, DetectorBase):
+    port_name = Cpt(Signal, value='NSLS2_EM')
```

Comment:
Mind leaving a comment here as to why this is necessary?

It's also a bit concerning that the QuadEM IOC is re-used for other devices than the NSLS2-EM. I don't know what their port names are, nor if they actually have an EPICS PV in their databases with the port name exposed...